docs/lessons.md

```diff
@@ -16,6 +16,12 @@
   "description": "Use the sprite editor to make player, tile, and enemy images",
   "url": "/lessons/barrel-dodger",
   "imageUrl": "/static/lessons/barrel-dodger.png"
+},
+{
+  "name": "Dance Party",
+  "description": "Create a basic dance collision game!",
+  "url": "/lessons/dance-party",
+  "imageUrl": "/static/lessons/barrel-dodger.png"
```

Comment:
left barrel dodger here instead of your own image ~

Also if you want you can set a ``largeImageUrl`` with a gif so that it will show up when selected / expanded e.g. [here](https://github.com/microsoft/pxt-arcade/pull/1121/files#diff-5f5bf66a6a610d5b34243b6a5a51cd02R15)